docs/advisors/checks/mysql_configuration_max_connections_usage.md

```diff
@@ -0,0 +1,12 @@
+# mysql_configuration_max_connections_usage
```

Comment:
@kedarvj, please suggest a human-friendly title here. Thanks!

Reply:
Does this sound OK?
```MySQL max connection usage check```